src/main/java/org/folio/rest/repository/FeeFineActionRepository.java

```diff
@@ -134,24 +134,41 @@ public FeeFineActionRepository(Map<String, String> headers, Context context) {
   public Future<List<Feefineaction>> findActionsByTypeForPeriodAndOwners(Action typeAction,
     String startDate, String endDate, List<String> ownerIds, int limit) {
 
+    final String startDateParameter = "actions.jsonb->>'dateAction' >= $4";
+    final String endDateParameter = "actions.jsonb->>'dateAction' < $5";
+    final String actionsParameter ="WHERE actions.jsonb->>'typeAction' IN ($2, $3)";
+
     String ownerIdsFilter = buildOwnerIdsFilter(ownerIds);
+    Tuple params = null;
```

Comment:
What if this null makes it through all of our `if-else`s? For example, if someone calls this method without knowing that `startDate==null && endDate!=null` is not acceptable? Will this cause any issues?